azurelinuxagent/common/event.py

```diff
@@ -237,8 +236,7 @@ def save_event(self, data):
         if len(existing_events) >= 1000:
             # To prevent the chance of #1035 happening - introduced periodic_WARNs.
             # More details about it - azurelinuxagent.common.event.add_log_event's pydoc.
-            logger.periodic_warn(logger.EVERY_FIFTEEN_MINUTES,
-                                 "[PERIODIC] Too many files under: {0}, current count:  {1}, removing oldest".format(
+            logger.warn("Too many files under: {0}, current count:  {1}, removing oldest".format(
```

Comment:
Can you make this periodic warn for 1 min (since we send telemetry every min). The reason is is the events dir > 1000, we would just keep writing this warn until we clear it up which would be at the end of the min.

Reply:
Good idea. Done. This could be caused by external events as well, which we won't have any control over.